craft_parts/parts.py

```diff
@@ -175,6 +175,18 @@ def has_overlay(self) -> bool:
             or self.overlay_files != ["*"]
         )
 
+    @property
+    def has_slices(self) -> bool:
+        """Return whether the part contains chisel slices."""
+        return any(p for p in self.stage_packages if "_" in p)
+
+    @property
+    def has_chisel_as_build_snap(self) -> bool:
+        """Return whether the part has chisel as build snap."""
+        return any(
```

Comment:
can't we just do 

` p for p in self.build_snaps if p.startswith("chisel")`

Reply:
No, because if there was a snap called "chiselhelper" for instance, it will match with that, which we don't want.

Reply:
yeah true